jujugui/static/gui/src/app/views/topology/service.js

```diff
@@ -189,6 +189,7 @@ YUI.add('juju-topology-service', function(Y) {
           'stroke': '#19b6ee',
```

Comment:
Can you add code comments to this effect for this and the one above so that we don't have to investigate this again in the future.